fhir-validation/src/test/java/com/ibm/fhir/validation/test/ExplanationOfBenefitTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.validation.test;
+
+import java.io.InputStream;
+
+import com.ibm.fhir.model.format.Format;
+import com.ibm.fhir.model.generator.FHIRGenerator;
+import com.ibm.fhir.model.parser.FHIRParser;
+import com.ibm.fhir.model.resource.Bundle;
+import com.ibm.fhir.model.resource.Bundle.Entry;
+import com.ibm.fhir.model.type.BundleType;
+import com.ibm.fhir.model.resource.ExplanationOfBenefit;
+import com.ibm.fhir.validation.FHIRValidator;
+
+public class ExplanationOfBenefitTest {
+    public static void main(String[] args) throws Exception {
+        try (InputStream in = ExplanationOfBenefitTest.class.getClassLoader().getResourceAsStream("JSON/explanationofbenefit.json")) {
```

Comment:
you could just use ExamplesUtil to get a reader for this spec example and use that.  no need to copy it into this project

Reply:
Per our conversation on Slack, this one isn't a spec example.